_about/notes/index.md

```diff
@@ -25,7 +25,3 @@ toc: false
   ```
   curl -L https://git.io/getIstio | sh -
   ```
-
-We typically wait to 'bake' the latest release for several weeks and ensure it is more stable than the previous one before promoting it to stable.
```

Comment:
this one can be replaced with some text and link to https://github.com/istio/istio/tree/master/release#istio-release